### ISSUE TITLE
Copy/Paste of TextFields

### ISSUE THREAD

**Issue by S-Man42** [S-Man42/GCWizard#112]
Currently the internal clipboard is only filled by the copy button. Add ability to insert copied content directly from text fields.

Same for pasting content: When pasting data through standard text field tool bar, open the internal clipboard popup menu.

Needs a custom selection toolbar: https://stackoverflow.com/questions/61998560/overwrite-paste-event-for-textformfield

**Comment by S-Man42** [comment]
https://github.com/justinmc/flutter-text-selection-menu-examples/blob/master/lib/custom_menu_page.dart

**Comment by S-Man42** [comment]
https://ktuusj.medium.com/flutter-custom-selection-toolbar-3acbe7937dd3

**Comment by S-Man42** [comment]
I tried but failed.. (.txt rename to .dart!)
[gcw_textselectioncontrols.txt](https://github.com/S-Man42/GCWizard/files/7228945/gcw_textselectioncontrols.txt)

See ll. 191ff: I tried to add a PopupMenu. However, for some reasons, when the callback comes back to l. 53, everything's fine but the selection. The "delegate" returns a selection of -1. When you choose simple "Paste" instead, delegate returns correct selection.

It works when you choose ll. 197ff. instead. So, there's something with the GCWPasteButton which let's the delegate forget its selection while the TextSelectionToolbarTextButton does not. Obvious main difference is, that TextSelectionToolbarTextButton is Stateless while GCWPasteButton is Stateful. Maybe something with that...

If you try, don't forget to escape the two setState() calls in GCWPasteButton._buildMenuItems. They don't work in the context. It should be checked if they are necessary anyway. As far as I saw, the "onSelected" calls of every caller inserted their on setState() calls...
